src/scripts/init.ts

```diff
@@ -56,36 +56,38 @@ interface MqttTeamDevice extends CertificateCredentials {
 }
 
 interface ContextInfo {
-    mqttTopicPrefix: string;
-    mqttTeamDeviceKeySSMParam: string;
-    mqttTeamDeviceCertSSMParam: string;
     nrfCloudMqttEndpoint: string;
-    mqttEndpoint: string;
+    mqttTopicPrefix: string;
     mqttTeamDeviceClientId: string;
-    localIotClientKeySSMParam: string;
+    mqttTeamDeviceClientIdSSMParam: string;
+    mqttTeamDeviceCertSSMParam: string;
+    mqttTeamDeviceKeySSMParam: string;
+    localMqttEndpoint: string;
     localIotClientCertSSMParam: string;
+    localIotClientKeySSMParam: string;
 }
 
 async function initializeContext(input: CliInput) {
     // Verify context file is valid
     await getExistingContext()
     const accountInfo = await getAccountInfo(input);
-    const iotEndpoint = (await Iot.send(new DescribeEndpointCommand({endpointType: "iot:Data-ATS"}))).endpointAddress
-    console.log(`AWS IoT endpoint: ${iotEndpoint}`)
+    const localIotEndpoint = (await Iot.send(new DescribeEndpointCommand({endpointType: "iot:Data-ATS"}))).endpointAddress
+    console.log(`Local AWS IoT endpoint: ${localIotEndpoint}`)
     console.log("Retrieved nRF Cloud account info:")
     console.log(JSON.stringify(accountInfo, null, 2))
-    await ensureNrfCloudCredentials(input);
-    await saveLocalIotClientCredentialsToSSM(input);
+    const mqttTeamDeviceClientId = await ensureNrfCloudMqttTeamDevice(input);
```

Comment:
Need the team device ID to put it into the mosquitto config file as the client ID, was hardcoded to `account-<tenantId>`